internal/authentication/file_user_provider.go

```diff
@@ -95,6 +111,10 @@ func (p *FileUserProvider) CheckUserPassword(username string, password string) (
 		}
 		return ok, nil
 	}
+
+	// TODO: Remove this. This is only here to temporarily fix the username enumeration security flaw in #949.
+	_, _ = CheckPassword(password, p.fakeHash)
```

Comment:
Panic on error here just to catch an issue with the generation of the hash in the future.